__tests__/Mock.test.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Mock from "../src/mock";
+import Enzyme from "./enzymeConfig";
```

Comment:
please Ndifreke, we are using resolvers for our imports. Checkout how I did my imports. This will mean you would have to update web pack with the aliases.

Reply:
I think resolver works in the domain of web pack, Jest and Enzyme cannot use the resolved names Webpack. Please correct me